packages/graphql/src/schema/resolvers/subscriptions/subscribe.ts

```diff
@@ -56,9 +57,18 @@ export function generateSubscribeMethod({
         if (entities.length) {
             const concreteEntity = entities[0] as ConcreteEntity;
             const hasAuthentication = concreteEntity.annotations.authentication;
-            if (hasAuthentication && hasAuthentication.operations.includes("SUBSCRIBE")) {
+            if (
+                hasAuthentication &&
+                (hasAuthentication.operations.includes("SUBSCRIBE") || hasAuthentication.jwtPayload)
```

Comment:
Shouldn't this be `&&` also? Looks like this would check authentication even if "SUBSCRIBE" isn't specified?

```suggestion
                (hasAuthentication.operations.includes("SUBSCRIBE") && hasAuthentication.jwtPayload)
```

Reply:
ah yes, I think the second part of the OR condition should be eliminated as it's checked further down in line 67. I don't think it should be an AND because jwtPayload does not have to be specified necessarily